src/main/java/com/currencyfair/onesignal/model/notification/Notification.java

```diff
@@ -724,6 +724,9 @@
     @JsonProperty("isChrome")
     private Boolean chrome;
 
+    @JsonProperty("external_id")
```

Comment:
Other `@JsonProperty` annotated fields have comments explaining their usage. Recommend writing a comment matching the format of the other fields in this file.

Reply:
Description looks good!